anda/tools/butler-bin/butler-bin.spec

```
@@ -0,0 +1,36 @@
+Name:           butler-bin
+Version:        15.24.0
+Release:        1%?dist
+Summary:        Command-line itch.io helper.
+URL:            https://itch.io/docs/butler
+Source0:        https://broth.itch.ovh/butler/linux-amd64/LATEST/archive/default
+Source1:        https://raw.githubusercontent.com/itchio/butler/refs/heads/master/LICENSE
+Source2:	https://raw.githubusercontent.com/itchio/butler/refs/heads/master/README.md
+License:        MIT
+Provides:  	itchio-butler-bin
+Provides: 	itch-butler-bin
+
```

the binary is probably x86 only right? If so, should add this

```suggestion
ExclusiveArch: x86_64
```